pkg/openstack/models/models.go

```diff
@@ -4,6 +4,34 @@
 
 package models
 
+import (
+	"time"
+
+	"github.com/uptrace/bun"
+
+	coremodels "github.com/gardener/inventory/pkg/core/models"
+	"github.com/gardener/inventory/pkg/core/registry"
+)
+
+// Server represents an OpenStack Server.
+type Server struct {
```

Comment:
Model does not seem to be aligned with the schema, please check my previous comment on this one.